src/app/services/ws.service.ts

```diff
@@ -350,4 +354,19 @@ export class WebSocketService {
       this.window.location.reload();
     });
   }
+
+  private getLoggedInUserInformation(): void {
```

Comment:
All this logic doesn't really belong to websocket service, but it's okay for now as this service is going away anyway.

Reply:
we can create separate store for it if needed.

Reply:
I thought if we handle `loginCallback` , `loginWithToken` and other things in here, so it will be OK to get user data here.